### PR TITLE
Update documentation to tell developers to use 1.9.4 version of ansible.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: en/developer_guide/install_with_vagrant.rst [hunk 1]
```diff
@@ -22,18 +22,11 @@ Install ansible
...
-    $ cd ./ansible
-    $ git checkout -b release1.8.2 origin/release1.8.2
-    $ git submodule init
-    $ git submodule update --recursive
-    $ source ./hacking/env-setup
+Our configuration was made for the 1.9.4 version of ansible.
 
-Go on the project page for more information : http://www.ansible.com
+For more information about ansible installation, you can go on ansible documentation
+page : http://docs.ansible.com/ansible/intro_installation.html
```
Comment (nicolasThal) [inline en/developer_guide/install_with_vagrant.rst]: Could you make a proper rst link ?

Comment (ghost) [inline en/developer_guide/install_with_vagrant.rst]: @nicolasThal done.

**Comment by nicolasThal** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

**Pull request merged**